charts/netdata/templates/ingress.yaml

```diff
@@ -26,8 +27,18 @@ spec:
       http:
         paths:
           - path: {{ $ingressPath }}
+            {{- if not (eq $apiVersion "extensions/v1beta1" "networking.k8s.io/v1beta1") }}
+            pathType: Prefix
```

Comment:
Actually, we can move `pathType` to the else branch like @chrisDeFouRire did in https://github.com/netdata/helmchart/pull/213/files